CLAUDE.md

```
@@ -0,0 +1,72 @@
+```bash
+npm run compile    # prisma generate && tsc && prisma migrate deploy
+npm start          # compile してからサーバー起動
+npm test           # Vitest でテスト実行
+npm run lint       # ESLint
+npm run fix        # ESLint 自動修正
+npm run create-migrate  # Prisma マイグレーションファイル作成
+```
+
+## ディレクトリ構成
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)